src/contexts/JsonRpcContext.tsx

```diff
@@ -539,6 +540,41 @@ export function JsonRpcContextProvider({
         };
       }
     ),
+    testSignMessage: _createJsonRpcRequestHandler(
+      async (
+        chainId: string,
+        address: string
+      ): Promise<IFormattedRpcResponse> => {
+        const method = DEFAULT_HEDERA_METHODS.HEDERA_SIGN_MESSAGE;
+
+        const params = {
+          message: {
+            bytes: Buffer.from(
```

Comment:
Do we want this to be `message.bytes`? Would a more generic `message.data` be more appropriate?

Reply:
yeah I guess it could be a string instead. would just need to be handled by the wallet as such as the `.sign` method takes a `UInt8Array`

Reply:
I think it would be best if it was sent as a base64 encoded string, just as how we did it for the transaction. I can make that change pretty easily.

Reply:
Oh, you were actually sending it as string already. I wonder how it actually worked on the wallet side then... 🤔 Will check it out